lib/permutations.rb

```diff
@@ -1,4 +1,8 @@
 
 def permutations?(string1, string2)
```

Comment:
Similar to the last one, you have a time complexity that can be improved with a hash.